core/sql/common/dfs2rec.h

```diff
@@ -186,9 +177,9 @@
   #define REC_BLOB                     160      // SQ only: blob datatype
   #define REC_CLOB                     161      // SQ only: clob datatype
 
+  #define REC_BOOLEAN                  170
```

Comment:
Is Boolean considered a numeric type? I'm assuming not, since 170 is not in the normal range for numeric types.